samples/matter/smoke_co_alarm/Kconfig

```diff
@@ -5,6 +5,16 @@
 #
 mainmenu "Matter nRF Connect Smoke CO Alarm Example Application"
 
+config SYSTEM_OFF
```

Comment:
would prefer this to have a prefix e.g. `APPLICATION_SYSTEM_OFF` because without a prefix, it just seems like this could easily conflict with something in zephyr in future